storybookjs/storybook
# Components: Add outline icon

## What I did
Added an outline icon to the `@storybook/components` icon set using the design provided by @domyen 

## How to test

- Is this testable with Jest or Chromatic screenshots?
Yes, the icon stories will automatically include this new icon

- Does this need a new example in the kitchen sink apps?
No

- Does this need an update to the documentation?
No

If your answer is yes to any of these, please make sure to include it in your PR.
